src/components/Public/Portal.js

```diff
@@ -54,22 +63,21 @@ const Container = styled.div`
 
 const ContentContainer = styled.div`
```

Comment:
Would be good if you could add some margins inside this container:

```
margin-right: 3vw;
margin-left: 3vw;
```

Reply:
Actually ignore the above – look at how the login margins/padding were implemented and try make something similar:
https://github.com/lumi-io/postings/blob/main/src/components/Public/Login.js